octavia_f5/restclient/as3objects/service.py

```diff
@@ -80,6 +82,12 @@ def get_esd_entities(servicetype, esd):
         else:
             service_args['profileTCP'] = 'normal'
```

Comment:
Not sure why I did it, but during testing I noticed that every ESD is disabling fastL4 because of this (the code for TCP fallback checks for a profileTCP)

Removing this else branch seems to work fine for me.